### ISSUE TITLE
Add cloud files not working

### ISSUE THREAD

**Issue by julesies** [samvera/hyrax#2804]
Several Cloud Related bugs have been reported and I can't tell what is a Nurax set up issue, a hyrax issue, or a Browse-Everything issue. I thought I'd report them all here and write up to the appropriate place as needed. I know there are BE PRs that may address some of these. 

1. Import from google drive doesn't work at all but prompts user "the app isn't verified" https://github.com/curationexperts/nurax/issues/217
2. Import from dropbox allows authenticated to client via hyrax but then prompts users to enter a code into nurax: https://github.com/curationexperts/nurax/issues/230
3. Import from box successfully imports but strips the file extension and so the user downloads a file that is confusing/generic. https://github.com/curationexperts/nurax/issues/233

**Comment by vantuyls** [comment]
I'm still looking for a little clarity here - @jrgriffiniii, do we expect BE to work for hyrax currently (or in the impending release)? we are hoping not to block the release based on upstream (ie BE) problems, but if the issues we're seeing are not BE related, we'll need to fix them before release. Can you help us tease this apart?

**Comment by jrgriffiniii** [comment]
@vantuyls this is definitely an upstream problem, and I would please recommend that any impending Hyrax 2.x releases *not* be blocked by Dropbox issues in `browse-everything`.

**Comment by vantuyls** [comment]
thanks @jrgriffiniii

**Comment by julesies** [comment]
@jrgriffiniii what about box and google drive? i believe box works, but there is a hyrax bug that strips out the file extension. and i'm not sure the status of google drive.

**Comment by vantuyls** [comment]
@jrgriffiniii can you give us an update on this issue and a status of any work that is ongoing or planned? @no-reply

**Comment by no-reply** [comment]
Anyone know where we are on this one? Did various BrowseEverything upgrades resolve any portion of this?

**Comment by rjkati** [comment]
In Hyrax 3.0.0 rc3, only Box can be selected as a cloud provider.
![box](https://user-images.githubusercontent.com/32644254/106188569-25e89700-6175-11eb-9477-06ba467e8928.PNG)

I am able to connect my Box account and select a file to upload ("Welcome to Box.pdf"). When I click "Submit", the "Submit" button displays a "Loading..." message and a spinning circle appears. The file does not upload and clicking the "Cancel" button does nothing. 
![spinnybox](https://user-images.githubusercontent.com/32644254/106189787-bf647880-6176-11eb-9220-ff782c0db028.PNG)

This occurs in both Chrome 88.0.4324.104 and Firefox 85.0. @jlhardes should this be a new ticket?

**Comment by jlhardes** [comment]
I think nurax-dev isn't set up to connect to various cloud providers, only to Box. And that setup might not be complete either. @jrgriffiniii, it looks like all services should be working (https://github.com/samvera/browse-everything) - is this more about inquiring with nurax-dev regarding what is configured for cloud upload?

**Comment by cudevmaxwell** [comment]
@rjkati We're also seeing the same behaviour for the file system provider. browse-everything 1.1.2, hyrax master branch head. Might need to be a new issue at the browse-everything repo?

**Comment by wickr** [comment]
We're seeing this as well on ScholarsArchive, on Hyrax 2.9.4.

https://github.com/osulp/Scholars-Archive/issues/2215

The plugin works and passes the file location to Hyrax, but won't close the modal. In our ticket is a workaround, which is to manually remove the browse-everything elements after submitting. After doing that, you can submit the file.

**Comment by jlhardes** [comment]
This looks like it might be something to report over in browse-everything for a fix?

**Comment by jlhardes** [comment]
Rutgers' recent implementation of browse-everything didn't have any issues with any cloud providers. We think recent work in browse-everything by @no-reply might have addressed the issue described here, can you verify?

**Comment by jlhardes** [comment]
Rutgers is using Hyrax 2.x and the latest browse-everything release does not include @no-reply's recent changes so we might need to check in with the community to see if we have an issue to work on here still for browse-everything with Hyrax 3.x.

**Comment by fritzfreiheit** [comment]
See: https://github.com/samvera/browse-everything/issues/376
